B.READ/B.READ/Sources/Presentation/Common/ValueObject/RecordCellVO.swift

```diff
@@ -85,6 +85,15 @@ extension RecordCellVO: Hashable {
   }
   
   static func == (lhs: RecordCellVO, rhs: RecordCellVO) -> Bool {
-    return lhs.id == rhs.id
+    if lhs.id != rhs.id { return false }
+    if lhs.readingState != rhs.readingState { return false }
+    if lhs.heart != rhs.heart { return false }
+    if lhs.progress != rhs.progress { return false }
+    if lhs.star != rhs.star { return false }
+    if lhs.memoCount != rhs.memoCount { return false }
+    if lhs.quoteCount != rhs.quoteCount { return false }
+    if lhs.period != rhs.period { return false }
+    if lhs.isFavorite != rhs.isFavorite { return false }
+    return true
```

Comment:
```suggestion
   return lhs.id == rhs.id &&
       lhs.readingState == rhs.readingState &&
       lhs.heart == rhs.heart &&
       lhs.progress == rhs.progress &&
       lhs.star == rhs.star &&
       lhs.memoCount == rhs.memoCount &&
       lhs.quoteCount == rhs.quoteCount &&
       lhs.period == rhs.period &&
       lhs.isFavorite == rhs.isFavorite
```
p3) 한줄한줄 고생하셨네요, 이렇게 쓰면 더편하게 쓸 수 있습니다. 하하

Reply:
디버그하거나 할때는 if 방식은 하나씩 디버그를 걸어볼 수 있어서 이렇게 작성했었는데!!
지금 봤을때는 하나씩 디버그까지는 필요없을거 같아서 수정했습니다!